### PR TITLE
Task/web 9

### PR DESCRIPTION
**Pull request by ilobo22**
The only problem I see is that when resizing the website, the additional menu tab resizes the menu bar. I will attach a picture to show what I'm talking about.
Screen Shot 2019-06-28 at 10 41 46 AM

### CODE REVIEW

**Comment by jdamerow** [comment]
please resolve conflicts and don't commit .DS_Store files

File: gallery/portfolio-item.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <i class='icon-layout'></i>
+          </a>
+          <a href='' title='next'>
+            <i class='icon-right-open'></i>
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class='four spacing'></div>
```
Comment (jdamerow) [inline gallery/portfolio-item.html]: where is this file used? It seems to only have dummy text in it?

File: gallery/portfolio.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    klass: Team
+  - image: /images/work.png
+    title: Serious Work
+    klass: Work
+  - image: /images/work2.png
+    title: Teamwork
+    klass: Work
+  - image: /images/team.jpg
+    title: The Pros
+    klass: Team
```
Comment (jdamerow) [inline gallery/portfolio.html]: is this used? it seems to have dummy text.

**Comment by ilobo22** [comment]
Taylor helped me merge and we think we found the conflict.

File: gallery/portfolio.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        </li>
+        {% endfor %}
+
+      </ul>
+
+    </div>
+  </div>
+
+  <div class='four spacing'></div>
+</div>
```
Comment (jdamerow) [inline gallery/portfolio.html]: let's make the images open in the same window as pop ups/with a grey backdrop or something; something like this:
image

**Comment by ilobo22** [comment]
Only problem I see is that if you scroll towards the bottom, the links overlap on top of the picture. Secondly, to exit out of the image we have an "X" at the top right.

**Comment by jdamerow** [comment]
great, just two more things:
- the menu bar should be great out as well.
- there should be forward/backward links to move through the images.

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by jdamerow** [comment]
The backdrop only goes to the bottom of the window and is missing if you scroll down. Also, let's put the forward/backward arrows on the image (left and right side in the middle).

image

**Comment by jdamerow** [comment]
the backdrop is still missing at the bottom if you scroll up.

**Comment by ilobo22** [comment]
By any chance, are you zoomed in? I've opened web-9 on safari and chrome on fullscreen and scrolling up doesn't affect the backdrop at all.

**Comment by jdamerow** [comment]
This is what happens now (and the window is almost the size of my screen):
image

**Comment by jdamerow** [comment]
Almost there. Only thing is to center the image in the middle of the window instead of giving it a set margin from the top. That just creates issues on smaller screens since it's pushed too far down and then cut off at the bottom.

**Pull request merged**